CONTRIBUTING.md

```diff
@@ -22,4 +22,35 @@ We do all of NeMo-text-processing's development in the open. Contributions from
 11) Optional: if you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
 12) Send your PR and request a review
 
+# Notes for Language Contribution
+1) `en/graph_utils.py` is the de facto parent for all other `graph_util` functions. Please refrain from duplicating code logics into new `graph_utils.py` files and default to imports from `en/graph_utils.py` instead. `LANG/graph_utils.py` files should only contain new methods and variables. Not all new languages will require a submodule specific `graph_utils.py` file.
 
+2) When writing taggers for semiotic classes, you may need to tag additional token properties (e.g. grammatical gender, case) for accurate verbalization. For example, the Spanish ordinal `21.º` carries masculine gender and is verbalized with a specific spelling. As such, it would be desired for the TN tagger to tokenize the string with gender the gender property included:
+
+`"21.º" -> ordinal { integer: "vigésimo primero" gender: "masc" }`
```

Comment:
let's switch "gender" to "morphosyntactic_features" to fit proto

Reply:
Mmm, I want to highlight the difference between what a user would be tempted to do and what they should do. Any suggestions to bring that out?